src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamprovider/StreamProvider.scala

```diff
@@ -49,6 +50,7 @@ class StreamProvider private(typeToFactories: Map[String, List[StreamFactory[_]]
     */
   def buildStream[A: TypeTag](
     streamingContext: StreamingContext,
+    configurationManager: ConfigurationManager,
```

Comment:
Can you pass this into the constructor of the Twitter stream factory instead? This change un-generalizes `StreamProvider`. It's fine to add this to `StreamProviderFactory` since that's not part of this re-usable package.

Reply:
Will do! Please see https://github.com/CatalystCode/project-fortis-spark/pull/128